Library/Homebrew/shims/linux/super/cc

```diff
@@ -184,6 +184,11 @@ class Cmd
       val  = chuzzle($1) || enum.next
       path = canonical_path(val)
       args << "-L#{val}" if keep?(path) && @lset.add?(path)
+    when /^-R(.+)?/
+      # -R is not the correct way to pass rpaths to the linker
+      val  = chuzzle($1) || enum.next
+      path = canonical_path(val)
+      args << "-Wl,-rpath=" if keep?(path) && @lset.add?(path)
```

Comment:
I'm okay with `keep?(path)`. It's useful to prevent adding wacky paths like `/usr/lib64` to the `RPATH`.

> gcc is often an undeclared runtime dependency of formulae built with gcc; and I think filtering paths here would change how formulae are currently built.

Currently we add neither `glibc` or `gcc` to the RPATH of formula, meaning that those two formulae have to be linked, or everything breaks. That's a separate issue though.